engine/src/main/java/org/terasology/rendering/logic/LightComponent.java

```diff
@@ -43,14 +44,29 @@
     @Replicate
     public float lightAmbientIntensity = 1.0f;
 
+    /**
+     * This helps control how focused the specular light is. A smaller number will make a wider cone of light. A larger number will make a narrower cone of light.
+     */
     @Replicate
-    public float lightSpecularPower = 1.0f;
+    public float lightSpecularPower = 100.0f;
+    /**
+     * Light attenuation range used in the calculation of how light fades from the light source as it gets farther away.  It is use in the following calculation:
+     * <p>
+     * attenuation = 1 / (lightDist/lightAttenuationRange + 1)^2
+     * <p>
+     * Where lightDist is how far the point in the world is from the light source.
+     */
     @Replicate
-    public float lightAttenuationRange = 16.0f;
+    public float lightAttenuationRange = 10.0f;
```

Comment:
Seems like an improvement to me, but only because I can see a little more! It's really hard to judge I guess because of the interplay between the lighting model per se and the perceptual system. If one lights up a torch in a dark place at the beginning it will blind you, something that could be achieved with exposure/contrast/bloom. But eventually, and especially if you shield the light source somehow, the rest of the environment would get quite visible. With this I do not want to detract from this PR. I'm merely pointing out that we'll probably have too look a little bit more holistically at the whole lighting situation, taking in account both the "simulated physics" or lights in Terasology and then how we want the "simulated eye" to respond to the various situation, something that is done in the post-processing steps of the rendering pipeline.